TWLight/users/templates/users/available_collection_tile.html

```diff
@@ -1,6 +1,13 @@
 {% load i18n %}
 <div class="col-xl-3 col-lg-4 col-md-6 col-sm-12">
   <div class="card collection-tile">
+    {% if available_collection.searchable == searchable %}
+      {% comment %}Translators: An icon that indicate whether a partner is searchable or not. {% endcomment %}
```

Comment:
Probably worth clarifying that this is hovertext for the icon - describing the text itself as an icon might be confusing, e.g.:

> Hovertext for an icon which indicates whether a collection is searchable or not.

Reply:
Done!